src/sardana/macroserver/macro.py

```diff
@@ -2238,6 +2238,12 @@ def parent_macro(self):
         internally by the *Executor*"""
         return self._parent_macro
 
+    @property
+    def _command(self):
```

Comment:
What should this be called? Just `command` feels dangerous, big risk of clashing with something in some macro. And the underscore doesn't match the rest of the unofficial api.
A few other alternatives: `command_string`, `macro_command`, `formatted_command`...